realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -862,21 +798,6 @@ public void run() {
 
     @Test
     @RunTestInLooperThread
-    public void realmListenerAddedAfterCommit() {
```

Comment:
Out of curiousity, what is the use-case for adding the RealmChangeListener **after** the transaction?

I haven't seen anyone do that before for any particular purpose.